docs/API.md

```diff
@@ -182,6 +182,23 @@ update msg model =
       (model, Ports.Router.routerLaunchRoute "ItemDetailsModal")
 ```
 
+
+#### `routerSetLocation`
+
+```elm
+port routerSetLocation : String -> Cmd msg
+```
+
+Will set the location and load that new page. This will require the full url to work correctly.
+
+#### `routerSetLocationRelative`
+
+```elm
+port routerSetLocationRelative : String -> Cmd msg
```

Comment:
This is a cool idea. It made me wonder, "what if I just `window.location("/foo")` -- does that do the same?" And it does! (Tried in all browsers except IE.) So we can get rid of this I think.

Reply:
If you want to document that case though, we could in the `routerSetLocation` section.